### ISSUE TITLE
Clean, P1 and BTS Laser Emmiters does not have "falloff" set

### ISSUE THREAD

**Issue by LautaroL20** [BEEmod/BEE2-items#972]
![20161103023259_1](https://cloud.githubusercontent.com/assets/12244376/19956745/94aa92f4-a16e-11e6-9103-165eb3d639a5.jpg)

**Comment by ZombiePwner1** [comment]
This is the same as your other issue you should've just made a comment on ur other post https://github.com/TeamSpen210/BEE2-items/issues/873#issuecomment-258066658

**Comment by LautaroL20** [comment]
No, the other issue actually got its own commit that will close automatically once next release is out no matter what. That means leaving clean, P1, and BTS without a fix if i merge the post.

**Comment by ZombiePwner1** [comment]
. . . Fine then. Also i left a message for u on steam.

**Comment by Luke18033** [comment]
What exactly is the issue here, something with the switchable light limit?

**Comment by LautaroL20** [comment]
The problem here was that the Laser Emitters were causing lighting problems on the Observation Room after placing 6 of them, and ive read a comment from TeamSpen210 saying that this is to do due that the instance of the Emitters did not have enabled an option called "Falloff" that prevents this from happen.

**Comment by Luke18033** [comment]
I still don't get what the issue is, is it to do with the switchable light limit or not? The normal limit is 2 per face, so only having it occur after you placed 6 would be kind of weird.

**Comment by LautaroL20** [comment]
Ive tested this issue again with PRL35 since i finally builded a new PC and seems that the issue no longer happens, the Observation Room can now turn its light fine with 6 or even more laser emmiters:  ![Portal 1 Observation Room](https://user-images.githubusercontent.com/12244376/65827841-e3eece80-e26a-11e9-8dda-d37074ff65b2.jpg)

**Comment by TeamSpen210** [comment]
Not quite, P1 style is a special case. Because there's the orange area behind, I just have the light permanently on.
